client/home/lib/myteam/components/deleteteambutton/index.coffee

```diff
@@ -0,0 +1,27 @@
+kd = require 'kd'
+TeamFlux = require 'app/flux/teams'
+showError = require 'app/util/showError'
+require('./styl/deleteteambutton.styl')
+
+module.exports = class DeleteTeamButton extends kd.CustomHTMLView
```

Comment:
It is not even related with this pr but this button is used in another kd view, so that's why.